lib/dpul_collections_web/live/search_live.ex

```diff
@@ -133,8 +135,31 @@ defmodule DpulCollectionsWeb.SearchLive do
 
   def search_item(assigns) do
     ~H"""
+    <hr />
     <div class="item">
-      <div class="underline text-lg"><%= @item.title %></div>
+      <div class="flex flex-wrap gap-5 md:max-h-60 max-h-[20rem] overflow-hidden justify-center md:justify-start">
+        <img
+          class="h-[350px] w-[350px] md:h-[225px] md:w-[225px]"
+          src="https://picsum.photos/350/350/?random"
+        />
+        <img
+          class="h-[350px] w-[350px] md:h-[225px] md:w-[225px]"
+          src="https://picsum.photos/350/350/?random"
+        />
+        <img
+          class="h-[350px] w-[350px] md:h-[225px] md:w-[225px]"
+          src="https://picsum.photos/350/350/?random"
+        />
+        <img
+          class="h-[350px] w-[350px] md:h-[225px] md:w-[225px]"
+          src="https://picsum.photos/350/350/?random"
+        />
+        <img
+          class="h-[350px] w-[350px] md:h-[225px] md:w-[225px]"
+          src="https://picsum.photos/350/350/?random"
+        />
+      </div>
+      <h2 class="underline text-xl font-bold pt-4"><%= @item.title %></h2>
```

Comment:
Seeing this as an h2 makes me curious; I would expect h1 to be the name of the site and h2 to be the title of the page. which if we had that would make these h3. We don't have a page title in our design, but would it be good to have one hidden from screen for accessibility reasons?

Reply:
@hackartisan that's a good point. I don't think we need to use the <h1> on every page for the name of the site, but the title of the page is important. In this case, I will add an <h1 class="sr_only"> to the page. Each result can then stay as an <h2>.